spec/type_extensions/time_spec.cr

```diff
@@ -0,0 +1,29 @@
+require "../../spec_helper"
+
+describe "UUID column type" do
+  describe ".parse" do
+    it "casts a String successfully" do
```

Comment:
Maybe it could say what kind of string? Like `casts an ISO8601 string successfully`?